lib/types/multipart.js

```diff
@@ -169,6 +170,10 @@ function Multipart(boy, cfg) {
       else
         encoding = '7bit';
 
+      for (var h in header) {
```

Comment:
I think we should just iterate over `Object.keys(header)` to avoid traversing prototypes and the like.